src/robusta/core/sinks/slack/slack_sink_params.py

```diff
@@ -35,19 +33,45 @@ def validate_channel_override(cls, v: str):
                 )
                 raise ValueError(err_msg)
         return v
+    
+    def normalize_placeholder(cls, s):
+        def repl(match):
+            return cls.normalize_key_string(match.group(0))
+        
+        result = s.replace(ANNOTATIONS_PREF, '').replace(LABELS_PREF, '')
```

Comment:
Hey @levtomer66 
Do we also need to change here, and call it with only one of `ANNOTATIONS_PREF` or `LABELS_PREF` each time or it works fine as it is?

Reply:
This should be fine as it just clear the prefixes leaving a standalone '$' for the template substitute

Reply:
got it, thanks!